libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -29,32 +29,38 @@
 #include "config.h"
 
 /* MQTT packet types. */
-#define MQTT_PACKET_TYPE_CONNECT         ( ( uint8_t ) 0x10U ) /**< @brief CONNECT (client-to-server). */
-#define MQTT_PACKET_TYPE_CONNACK         ( ( uint8_t ) 0x20U ) /**< @brief CONNACK (server-to-client). */
-#define MQTT_PACKET_TYPE_PUBLISH         ( ( uint8_t ) 0x30U ) /**< @brief PUBLISH (bidirectional). */
-#define MQTT_PACKET_TYPE_PUBACK          ( ( uint8_t ) 0x40U ) /**< @brief PUBACK (bidirectional). */
-#define MQTT_PACKET_TYPE_PUBREC          ( ( uint8_t ) 0x50U ) /**< @brief PUBREC (bidirectional). */
-#define MQTT_PACKET_TYPE_PUBREL          ( ( uint8_t ) 0x62U ) /**< @brief PUBREL (bidirectional). */
-#define MQTT_PACKET_TYPE_PUBCOMP         ( ( uint8_t ) 0x70U ) /**< @brief PUBCOMP (bidirectional). */
-#define MQTT_PACKET_TYPE_SUBACK          ( ( uint8_t ) 0x90U ) /**< @brief SUBACK (server-to-client). */
-#define MQTT_PACKET_TYPE_UNSUBACK        ( ( uint8_t ) 0xB0U ) /**< @brief UNSUBACK (server-to-client). */
-#define MQTT_PACKET_TYPE_PINGRESP        ( ( uint8_t ) 0xD0U ) /**< @brief PINGRESP (server-to-client). */
-#define MQTT_PACKET_TYPE_DISCONNECT      ( ( uint8_t ) 0xE0U ) /**< @brief DISCONNECT (client-to-server). */
+#define MQTT_PACKET_TYPE_CONNECT       ( ( uint8_t ) 0x10U )   /**< @brief CONNECT (client-to-server). */
+#define MQTT_PACKET_TYPE_CONNACK       ( ( uint8_t ) 0x20U )   /**< @brief CONNACK (server-to-client). */
+#define MQTT_PACKET_TYPE_PUBLISH       ( ( uint8_t ) 0x30U )   /**< @brief PUBLISH (bidirectional). */
+#define MQTT_PACKET_TYPE_PUBACK        ( ( uint8_t ) 0x40U )   /**< @brief PUBACK (bidirectional). */
+#define MQTT_PACKET_TYPE_PUBREC        ( ( uint8_t ) 0x50U )   /**< @brief PUBREC (bidirectional). */
+#define MQTT_PACKET_TYPE_PUBREL        ( ( uint8_t ) 0x62U )   /**< @brief PUBREL (bidirectional). */
+#define MQTT_PACKET_TYPE_PUBCOMP       ( ( uint8_t ) 0x70U )   /**< @brief PUBCOMP (bidirectional). */
+#define MQTT_PACKET_TYPE_SUBACK        ( ( uint8_t ) 0x90U )   /**< @brief SUBACK (server-to-client). */
+#define MQTT_PACKET_TYPE_UNSUBACK      ( ( uint8_t ) 0xB0U )   /**< @brief UNSUBACK (server-to-client). */
+#define MQTT_PACKET_TYPE_PINGREQ       ( ( uint8_t ) 0xC0U )   /**< @brief PINGREQ (client-to-server). */
+#define MQTT_PACKET_TYPE_PINGRESP      ( ( uint8_t ) 0xD0U )   /**< @brief PINGRESP (server-to-client). */
+#define MQTT_PACKET_TYPE_DISCONNECT    ( ( uint8_t ) 0xE0U )   /**< @brief DISCONNECT (client-to-server). */
+
+/*
+ * Constant relating to PINGREQ and PINGRESP packets, defined by MQTT 3.1.1 spec.
+ */
+#define MQTT_PACKET_PINGREQ_SIZE       ( 2U )   /**< @brief A PINGREQ packet is always 2 bytes in size. */
```

Comment:
This is defined here but for disconnect and acks, they're defined in the .c file. They should all be in the same place for consistency.

Reply:
It is used in mqtt.c also. Hence it is not a private #define. That is the reason it is placed in the .h file instead of .c